src/components/PassageDetail/PassageDetailTranscribe.tsx

```diff
@@ -145,6 +158,27 @@ export function PassageDetailTranscribe({
     });
   };
 
+  const handleReopen = () => {
+    setStepComplete(currentstep, false);
+    if (hasChecking && nextStep) setStepComplete(nextStep, false);
+    if (curRole === 'editor' && prevStep) setStepComplete(prevStep, false);
+  };
+
+  const handleReject = (reason: string) => {
+    setStepComplete(currentstep, false);
+    if (hasChecking && nextStep) setStepComplete(nextStep, false);
+    if (curRole === 'editor' && prevStep) setStepComplete(prevStep, false);
```

Comment:
can you call handleReopen instead of copying the above 3 lines (or create a local function to do them)

Reply:
sure